int16 pwm_left
int16 pwm_right
---
int32 ticks_left
int32 ticks_right
